app/models/miq_schedule_worker/runner.rb

```diff
@@ -108,6 +108,14 @@ def schedules_for_all_roles
       :tags     => [:log_status, schedule_category]
     ) { enqueue :miq_server_worker_log_status }
 
+    # Schedule - Daily check for new CVE content for container images
+    every = worker_setting_or_default(:container_cve_check_interval, 1.day)
+    @schedules[:all] << system_schedule_every(
+      every,
+      :first_in => 1.hour, # Testing purposes
```

Comment:
On second thought... If you're testing, can't you use `:first_in => every` like above and set the container_cve_check_interval in your settings to 5.minutes or something small like that?